glm/CMakeLists.txt

```diff
@@ -42,8 +42,7 @@ source_group("SIMD Files" FILES ${SIMD_SOURCE})
 source_group("SIMD Files" FILES ${SIMD_INLINE})
 source_group("SIMD Files" FILES ${SIMD_HEADER})
 
-add_library(glm INTERFACE)
-target_include_directories(glm INTERFACE ../)
+include_directories(${CMAKE_CURRENT_SOURCE_DIR}/..)
```

Comment:
I would really recommend to use dependencies, and not use explicit include_directories. You can find the explanation in this [really great tutorial](https://pabloariasal.github.io/2018/02/19/its-time-to-do-cmake-right/#model-dependencies-with-target_link_libraries).